src/SourceBuild/content/eng/pipelines/templates/jobs/sdk-diff-tests.yml

```diff
@@ -47,16 +47,37 @@ jobs:
 
       echo "Sdk build: https://dev.azure.com/dnceng/internal/_build/results?buildId=$sdk_build&view=results"
 
-      echo "##vso[build.addbuildtag]sdk-$sdk_sha"
+      # TODO: Remove this once sdk starts producing the MSFT SDK - also remove installer tag from the builds
+      dotnet_dotnet_branch=$(az pipelines build show --organization '$(AZDO_ORG)' --project '$(AZDO_PROJECT)' --id $dotnet_dotnet_build --query "sourceBranch" --output tsv)
+      installer_build=$(az pipelines runs list \
+        --organization '$(AZDO_ORG)' \
+        --project '$(AZDO_PROJECT)' \
+        --pipeline-ids '$(INSTALLER_OFFICIAL_CI_PIPELINE_ID)' \
+        --query "[?sourceBranch == '$dotnet_dotnet_branch' && status == 'completed' && (result == 'succeeded' || result == 'partiallySucceeded')].id" \
+        --output tsv | head -n 1)

```

Comment:
This looks to be just getting the latest successful build of installer. It should be correlated with the commit that the dotnet/dotnet build is based on. If that's not possible given the current state of the migration, then I don't see much value in having this implementation. The results of the diff wouldn't necessarily be accurate.

Reply:
Thinking about this, I think Matt is right.  The differences between the SDK and installer builds are going to be significant until the end of release branches.  I think we are going to need to disable the tests until the tarballs are produces out of the sdk repo.